ERCS/erc-7786.md

```diff
@@ -74,6 +74,13 @@ interface IERC7786GatewaySource {
         bytes calldata payload,
         bytes[] calldata attributes
     ) external payable returns (bytes32 outboxId);
+
+    function requestRelay(
```

Comment:
are there any bridges that don't actually have a concept of a relayer? Maybe something in the l2 environment

Reply:
In L2 environments, as far as I've seen, relaying is present but in a different form where it's just built into the protocol and doesn't rely on third parties. But other than that it's very similar in the sense that you need to pay for execution.

Reply:
Going from mainnet to matic (polygon) I'm not sure I used to pay anything (other than the high gas cost of sending the message on mainnet)